packages/synapse-react-client/src/components/TableColumnSchemaEditor/TableColumnSchemaEditorUtils.ts

```diff
@@ -264,38 +263,6 @@ export function canHaveRestrictedValues(
   }
 }
 
-/**
- * Transform the form data for the TableColumnSchemaForm to ColumnModels to be sent to Synapse
- * @param formData the form data to transform
- */
-export function transformFormDataToColumnModels(
```

Comment:
The Zod schema we defined now does this transformation for us, so we can delete this function.